### ISSUE TITLE
Implement colour blind mode

### ISSUE THREAD

**Issue by kam56** [ItJustWorksBetterTM/EiffelVis#6]
As a user, I want to be able to change the theme into colour blind mode, so that I can distinguish nodes better.

**Acceptance Criteria**

- [ ] Frontend colours should be clear for a person with a deuteranomaly colour blindness.
- [ ] Settings to persist cookie over sessions.
- [ ] Find a colorblind person to test.
- [ ] Colorblind mode should be available via button press and keyboard shortcut.
- [ ] Code is proper typescript with type definitions for everything.
- [ ] Code is commented.

**Comment by alrapal** [comment]
Close this issue since it has been implemented as an acceptance criteria with #21
